server/test/services/nextcloud-talk/index.test.js

```diff
@@ -0,0 +1,45 @@
+const { expect } = require('chai');
+const NextcloudTalkService = require('../../../services/nextcloud-talk');
+
+describe('nextcloud-talk', () => {
+  it('should start service', async () => {
+    const gladys = {
+      variable: {
+        getVariables: () => Promise.resolve([{ user_id: 'f0de00a8-8ba7-4a4e-8f5a-7a21e94f36a8', value: 'testToken' }]),
+        getValue: () => Promise.resolve('testvalue'),
```

Comment:
Why don't you use fake for this?

fake.resolve() 

?

It let's you test that the function was indeed called

Example:

https://github.com/GladysAssistant/Gladys/blob/ee812a2afae3aeeabce870d3e607975a7a026d63/server/test/services/mqtt/lib/installContainer.test.js#L81